InSource/InSourceViewControl.cs

```diff
@@ -34,6 +35,16 @@ public InSourceViewControl()
 			_messageBoxService = new MessageBoxService();
 		}
 
+		public void ClearMessages()
+		{
+			_resultsTextBox.Text = "";
+		}
+
+		public void ReportMessage(FileBasedProject fileBasedProject, string message)
+		{
+			_resultsTextBox.AppendText("\r\n" + message);
```

Comment:
Can we use here string interpolation?

Reply:
Done. I've updated within the new commit.